template-parts/category-content.php

```diff
@@ -13,24 +13,25 @@
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 
 	<div class="grid-x grid-margin-x">
+		<?php $base_theme_medium_block = 'medium-12';
+		if ( has_post_thumbnail() ) { 
+			$base_theme_medium_block = 'medium-9';
+		?>
+			<div class="cell small-12 medium-3">
+				<?php
+				if ( ! is_single() && ! is_page() ) {
 
-		<div class="cell small-12 medium-3">
-			<?php
-			if ( ! is_single() && ! is_page() ) {
-
-				if ( has_post_thumbnail() ) {
-
-					the_post_thumbnail( 'large' );
+					if ( has_post_thumbnail() ) {
 
-				} else {
+						the_post_thumbnail( 'large' );
 
-					base_theme_placeholder_image();
+					}
 				}
-			}
-			?>
-		</div>
+				?>
+			</div>
+		<?php } ?>
 
-		<div class="cell small-12 medium-9">
+		<div class="cell small-12 <?php echo esc_html_e( $base_theme_medium_block ); ?>">
```

Comment:
@rashmi-sc Used `esc_html_e` which is wrong here, firstly it is a translation function, second you have already added echo then you don't need to use the `_e` function there.

Reply:
@rashmi-sc Also you have used `esc_html` where you are adding that as an HTML attribute.

Reply:
@rashmi-sc I have fixed these issues, please check those once you are back.